swim.go

```diff
@@ -444,9 +641,25 @@ func createPingMessage(mbrStatsMsg *pb.MbrStatsMsg) pb.Message {
 	}
 }
 
-func createNackMessage(seq string, mbrStatsMsg *pb.MbrStatsMsg) pb.Message {
+func createIndMessage(src, target string, mbrStatsMsg *pb.MbrStatsMsg) pb.Message {
```

Comment:
Is it better to create an id outside than to create it at random? It seems to be a problem when you try to build a test.

Reply:
This corresponds to all create functions.

Reply:
reasonable! 👍